gobblin-core/src/test/java/org/apache/gobblin/recordaccess/AvroGenericRecordAccessorTest.java

```diff
@@ -139,6 +139,11 @@ public void testGetStringArrayUtf8() throws IOException {
 
   @Test
   public void testGetMultiConvertsStrings() throws IOException {
+    // The below error is due to invalid avro data. Type with "null" union must have "null" first and then
```

Comment:
This comment is not strictly accurate (as far as Avro is concerned). A union with "null" doesn't _have_ to have null as the first type. It's usually intended to signify optional-ness, and usually people do want to put null first, so that they can set "null" as the default value, but it's not required.

What _is_ required (as per Avro) is that the default value must have the same type as the first entry in the union. So, for example, this is correct:
```
"type": ["null", "string"],
"default": null
```

Whereas this is wrong:
```
// default is null, but must be a string, since that's the first type in the union
"type": ["string", "null"],
"default": null
```

This is also wrong:
```
// default is a string ("null"), not null, which is the first type in the union
"type": ["null", "string"],
"default": "null"
```

And this is totally correct, though usually rare to see:
```
"type": ["string", "null"]
"default": "foobar"
```

So what you really want to say in this comment is that since the default value is `null`, you want to ensure that that's the first type in the union.